src/issue.ts

```diff
@@ -0,0 +1,105 @@
+import * as core from "@actions/core";
+import levenshtein from 'js-levenshtein';
+
+export interface IParameter {
+  area: string;
+  keywords: string[];
+  labels: string[];
+  assignees: string[];
+}
+
+export class Issue {
+  private title: string;
+  private body: string;
+  public parameters: IParameter[]
+  private similarity: number
+  private bodyValue: number
+
+  constructor(content: string[]) {
+    this.title = content[0];
+    this.body = content[1];
+    const excluded: string[] = core.getInput("excluded-expressions", {required: false}).replace(/\[|\]/gi, '').split('|');
+    excluded.forEach(ex => {
+      this.title.replace(ex, '');
+      this.body.replace(ex, '')
+    });
+    this.parameters = JSON.parse(core.getInput("parameters", {required: true}));
+    this.similarity = +core.getInput("similarity", {required: false})
+    this.bodyValue = +core.getInput("body-value", {required: false})
+  }
+
+  public determineArea(): string {
+    let titleIssueWords = this.title.split(/ |\./);
+    let bodyIssueWords = this.body.split(/ |\./)
+    let titleValue: number = 1
+    let x: number = 1;
+    let potentialAreas: Map<string, number> = new Map()
+      
+    // For each word in the title, check if it matches any keywords. If it does, add decreasing score based on inverse function to the area keyword is in.
+    titleIssueWords.forEach(content => {
+      potentialAreas = this.scoreArea(content, potentialAreas, titleValue);
+      ++x
+      titleValue = (2/(1+x))
```

Comment:
Careful of magic numbers! Store `(2/(1+x))` as a const modifier at the top level. This will make it easier to adjust later and makes the code generally more readable.

eg.

```ts
const weightedTitle: number = (n: number) => return (2/(1+n));

...

titleValue = weightedTitle(x)
```